setup.py

```diff
@@ -0,0 +1,86 @@
+# =================================================================
+#
+# Authors: Rob van Loon <borrob@me.com>
+#          Tom Kralidis <tomkralidis@gmail.com>
+#
+# Copyright (c) 2019 Rob van Loon
+#
+# Permission is hereby granted, free of charge, to any person
+# obtaining a copy of this software and associated documentation
+# files (the "Software"), to deal in the Software without
+# restriction, including without limitation the rights to use,
+# copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the
+# Software is furnished to do so, subject to the following
+# conditions:
+#
+# The above copyright notice and this permission notice shall be
+# included in all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
+# EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES
+# OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+# NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT
+# HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY,
+# WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+# FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
+# OTHER DEALINGS IN THE SOFTWARE.
+#
+# =================================================================
+import io
+from setuptools import find_packages, setup
+
+
+def read(filename, encoding='utf-8'):
+    """read file contents"""
+    with io.open(filename, encoding=encoding) as fh:
+        contents = fh.read().strip()
+    return contents
+
+
+def get_package_version():
```

Comment:
We should minimize the places where the version is specified.  Suggest:

- remove `VERSION`
- add function https://github.com/geopython/pygeoapi/blob/master/setup.py#L120-L127 to `setup.py`

Reply:
My bad. FIx with commit bb96eb0